templates/email/default/_email_sidebar.html

```diff
@@ -1,6 +1,19 @@
-[% DEFAULT report = object ~%]
+[%
+
+# There are cases (eg: when sending email about an update on a problem)
+# where `object` might not be a report. So in those cases, you can pass
+# in a `report` argument, which should be the report the email is about.
+# Otherwise, we assume the `object` you passed is a report.
+DEFAULT report = object ~%]
+
+# There are cases (eg: when confirming a not-yet-published report) where
```

Comment:
You have ended a `[% … %]` block just above, so all this is output in the email.